BackofficeBundle/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -284,6 +284,24 @@
     </div>
 {% endblock %}
 
+{% block oo_field_type_widget %}
+    {% spaceless %}
+        {{ form_row(form.fieldId) }}
+        {{ form_row(form.labels) }}
+        {{ form_row(form.type) }}
+        {{ form_row(form.listable) }}
+        {{ form_row(form.translatable) }}
+        {{ form_row(form.searchable) }}
+        {{ form_row(form.orderable) }}
+        {{ form_row(form.orderDirection) }}
+        <div class="form-group">
```

Comment:
As you want to group fields, you should maybe use:
        <fieldset>
        	<legend>{{ 'open_orchestra_backoffice.form.field_type.additionalParameters'|trans }}</legend>
        	{{ form_rest(form) }}
        </fieldset>